docs/flatcar-linux/packet.md

```diff
@@ -159,6 +132,19 @@ module "controller" {
   node_private_cidr = "10.128.156.0/25"
 }
 
+# DNS module that creates the required DNS entries for the cluster.
+# More details in DNS Zone section of this document.
+module "dns" {
+  source = "git::https://github.com/kinvolk/lokomotive-kubernetes//dns/route53?ref=<hash>"
```

Comment:
Those examples should be copy-pastable for the user, even if they point to some old commit hash or to the master branch.

Reply:
Currently `<hash>` is used, so it's okay to keep it consistent for now.